servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Single.java

```diff
@@ -898,6 +898,31 @@ public final Completable flatMapCompletable(Function<? super T, ? extends Comple
         return zip(this, other, zipper);
     }
 
+    /**
+     * Create a new {@link Single} that emits the results of a specified zipper {@link BiFunction} to items emitted by
+     * {@code singles}. All operations will terminate (even if there are failures) before the returned {@link Single}
+     * terminates.
+     * <p>
```

Comment:
Perhaps should say more about what the error terminations look like. What does `onError` receive if both `onError`? Either `onError`? Is the item lost when the other Single errors?

Reply:
I updated the javadoc. ptal.